packages/frontend/component/src/components/app-sidebar/sidebar-header/sidebar-switch.tsx

```diff
@@ -2,20 +2,41 @@ import { useAFFiNEI18N } from '@affine/i18n/hooks';
 import { SidebarIcon } from '@blocksuite/icons';
 import { IconButton } from '@toeverything/components/button';
 import { Tooltip } from '@toeverything/components/tooltip';
-import { useAtom } from 'jotai';
+import { useAtom, useSetAtom } from 'jotai';
+import { useEffect } from 'react';
 
-import { appSidebarOpenAtom } from '../index.jotai';
+import { appSidebarFloatingAtom, appSidebarOpenAtom } from '../index.jotai';
 import * as styles from './sidebar-switch.css';
 
 export const SidebarSwitch = ({ show }: { show: boolean }) => {
   const [open, setOpen] = useAtom(appSidebarOpenAtom);
+  const setAppSidebarFloating = useSetAtom(appSidebarFloatingAtom);
   const t = useAFFiNEI18N();
   const tooltipContent = open
     ? t['com.affine.sidebarSwitch.collapse']()
     : t['com.affine.sidebarSwitch.expand']();
   const collapseKeyboardShortcuts =
     environment.isBrowser && environment.isMacOs ? ' ⌘+/' : ' Ctrl+/';
 
+  useEffect(
+    () => {
+      const isMobile = (() => {
+        if (environment.isBrowser) {
+          return environment.isMobile;
+        }
+        return false;
+      })();
+      if (isMobile) {
+        // to make sure the sidebar is closed on mobile
+        setOpen(false);
+        // to make sure the sidebar is floating on mobile
+        setAppSidebarFloating(true);
+      }
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
```

Comment:
So we don't need to think about window resizing in desktop browsers?(e.g. from large to mobile size) 

@hwangdev97 cc

Reply:
I tried to resize the sidebar on the web version of our site using my mobile, and it seems like it's not working?

Reply:
We haven't designed the behavior of the mobile terminal yet. I think it can be technically convenient.